code/_onclick/telekinesis.dm

```diff
@@ -109,7 +109,7 @@
 	if(!host || host != user)
 		qdel(src)
 		return
-	if(!host.dna?.GetSEState(GLOB.teleblock))
+	if(!host.dna && HAS_TRAIT(host, TRAIT_TELEKINESIS))
```

Comment:
Small logic error here. The previous code was seeing if they *did not* have telekinesis. You are now checking if they  *do* have telekinesis. Should be:
```suggestion
	if(!host.dna && !HAS_TRAIT(host, TRAIT_TELEKINESIS))
```